### DISCUSSION TITLE
Summaries for Course 1, Our Privacy Opportunity

### DISCUSSION THREAD

**Discussion by JohannesStutz** [OpenMined/courses#227] [Resources]
Hi everyone, I wrote (rather long) summaries of the lessons in the first course. Maybe you already saw them on Slack, but here they are in case you ever want to look up a concept :)

- [**Lesson 2** Information flows, privacy-transparency tradeoff](https://deeplearning.berlin/online%20courses/privacy/the%20private%20ai%20series/openmined/2021/01/08/Our-Privacy-Opportunity-Part-1.html)
- [**Lesson 3** Copy problem, bundling problem, recursive enforcement problem](https://deeplearning.berlin/online%20courses/privacy/the%20private%20ai%20series/openmined/2021/01/11/Our-Privacy-Opportunity-Part-2.html)
- [**Lesson 4.1** Structured transparency, input and output privacy](https://deeplearning.berlin/online%20courses/privacy/the%20private%20ai%20series/openmined/2021/01/19/Our-Privacy-Opportunity-Part-3.html)
- [**Lesson 4.2** Input and output verification, flow governance](https://deeplearning.berlin/online%20courses/privacy/the%20private%20ai%20series/openmined/2021/01/20/Our-Privacy-Opportunity-Part-4.html)
- [**Lesson 5** The impact of structured transparency](https://deeplearning.berlin/online%20courses/privacy/the%20private%20ai%20series/openmined/2021/01/25/Our-Privacy-Opportunity-Part-5.html)

Thank you to the whole OpenMined team for this course! Just writing the summaries gave me a certain idea of how much work went into writing and producing these videos.

**Comment by anishpdalal** [comment]
Thanks these are very helpful!
